client/app/billing/payment/method/controller.js

```diff
@@ -0,0 +1,71 @@
+import { ALERTER_ID } from './constants';
+
+export default class BillingPaymentMethodCtrl {
+  /* @ngInject */
+  constructor($timeout, Alerter, OVH_PAYMENT_MEAN_STATUS, OVH_PAYMENT_METHOD_TYPE,
+    ovhPaymentMethodHelper) {
+    // dependencies injections
+    this.$timeout = $timeout;
+    this.Alerter = Alerter;
+    this.OVH_PAYMENT_MEAN_STATUS = OVH_PAYMENT_MEAN_STATUS;
+    this.OVH_PAYMENT_METHOD_TYPE = OVH_PAYMENT_METHOD_TYPE;
+    this.ovhPaymentMethodHelper = ovhPaymentMethodHelper;
+
+    // other attributes used in views
+    this.ALERTER_ID = ALERTER_ID;
+    this.tableFilterOptions = null;
+    this.guide = null;
+    this.hasPendingValidationBankAccount = false;
+  }
+
+  /* =====================================
+  =            INITIALIZATION            =
+  ====================================== */
+
+  $onInit() {
+    // TODO: create a component that handle message from ui router resolve
+    if (this.message) {
+      this.$timeout(() => {
```

Comment:
is the timeout necessary ? i hate them ^^

Reply:
Unfortunatly, yes :-(
If no `$timeout` I don't know what happen, but the Alerter is reseted...